src/index/fetcher.rs

```diff
@@ -65,18 +65,34 @@ impl Fetcher {
     }
 
     let body = Value::Array(reqs).to_string();
-    let req = Request::builder()
-      .method(Method::POST)
-      .uri(&self.url)
-      .header(hyper::header::AUTHORIZATION, &self.auth)
-      .header(hyper::header::CONTENT_TYPE, "application/json")
-      .body(Body::from(body))?;
 
-    let response = self.client.request(req).await?;
-
-    let buf = hyper::body::to_bytes(response).await?;
-
-    let mut results: Vec<JsonResponse<String>> = serde_json::from_slice(&buf)?;
+    let mut results: Vec<JsonResponse<String>>;
+    let mut tries = 0;
+
+    loop {
+      results = match self.try_get_transactions(body.clone()).await {
+        Ok(results) => results,
+        Err(error) => {
+          if tries >= 5 {
+            return Err(anyhow!(
+              "Failed to fetch raw transactions after 5 retries. Error: {}",
```

Comment:
```suggestion
              "failed to fetch raw transactions after 5 retries: {}",
```
We're not very consistent with not capitalizing error message ourselves but I want to try to do that. So if you could make all the error messages lowercase that would be great. Also I don't think we need the "Error:" just colon or semicolon and then the "{}". Thanks!